spec/cocoapods/generate/configuration_spec.rb

```diff
@@ -156,6 +158,42 @@ def should_raise(*args, &blk)
     end
   end
 
+  describe_class_method :podspecs_from_paths do
```

Comment:
new tests added here that were missing before but also test the new logic added.